packages/adp-tooling/src/preview/adp-preview.ts

```diff
@@ -14,7 +14,8 @@ import type { AdpPreviewConfig, DescriptorVariant } from '../types';
 
 export const enum ApiRoutes {
     FRAGMENT = '/adp/api/fragment',
-    CONTROLLER = '/adp/api/controller'
+    CONTROLLER = '/adp/api/controller',
+    CODE_EXT = '/adp/api/code_ext/:controllerName'
```

Comment:
Why not `/adp/api/code_ext/controllerName`?

Reply:
Because I am passing controllerName in the params and retrieving it like this:
![image](https://github.com/SAP/open-ux-tools/assets/39880364/a2d75873-0893-415c-aa24-3e7adc7b6aeb)

And instead of using a POST request I am using GET request with a dynamic parameter. And if I use this `/adp/api/code_ext/controllerName` it means I have to send a body and use POST request, but the data in my case is just a string. And the dynamic parameter indicates that I am accessing a controller with a specific name.

Reply:
Thanks, I overlooked that `:controllerName` is a dynamic parameter.